src/components/Postiterie.vue

```diff
@@ -238,26 +243,51 @@ export default {
       xhr.responseType = 'json';
       xhr.onload = () => {
         if (xhr.response.id){
+          
           this.link = 'https://drive.google.com/file/d/' + xhr.response.id + '/view';
+
+          //This generates a direct link to drawIO, but the file needs to be manually opened with drawIO first
+          //let profileId = gapi.auth2.getAuthInstance().currentUser.get().getId();
+          //this.link = 'https://www.draw.io/?state={"ids":["' + xhr.response.id + '"],"action":"open","userId":"' + profileId + '"}' + '#G' + xhr.response.id
+
           //if email input, send mail else redirect to the file
           if (this.email){
-              this.prepareMail();
+            this.prepareMail();
           }
           else{
-              window.location.href = this.link;
+            window.location.href = this.link;
           }
         }
       };
       xhr.send(form);
     },
 
+    getImageFromScreenshot: function(image){
+      this.camera = false;
+      this.image = image.src; 
+    },
+
     // function prepareMail 
     // TODO
     // no IO
     // call the mail API or our mail server
     prepareMail: function(){
-        console.log("prepareMail was fired");
+        //  Using smtpjs to send mail
+        // TODO : smtpjs can encrypt credentiel and return a token but we need to pass a domain name so come back here for deplyment
+        Email.send({
+          Host : "smtp.gmail.com",
+          Username : process.env.VUE_APP_SMTP_EMAIL,
```

Comment:
You probably don't want SMTP email and password to appear in clear in the compiled output in everyone's browser.

Reply:
SmtpJS can generate a security token that will allow us to remove our credentials from the script. 
I've set it up this way for now as the feature requires a domain name